### DISCUSSION TITLE
Musikcube

### DISCUSSION THREAD

**Discussion by EdenQwQ** [catppuccin/catppuccin#1947] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
[my port](https://github.com/EdenQwQ/musikcube)

![screenshot](https://github.com/EdenQwQ/musikcube/blob/main/assets/screenshot.png?raw=true)

**Comment by Dukeatron** [comment]
Greetings Eden

Your port is welcome, but there are several issues with your repository. Please read [CONTRIBUTING.md](https://github.com/catppuccin/catppuccin/blob/main/CONTRIBUTING.md) for proper repository preparation
